src/relay/environment.js

```diff
@@ -1,15 +1,30 @@
 // @noflow
 
-import { Environment, Network, RecordSource, Store } from 'relay-runtime';
-
+import {
+  Environment,
+  Network,
+  RecordSource,
+  Store,
+  QueryResponseCache,
+} from 'relay-runtime';
 import { getSessionToken } from '../helpers/Auth';
 
 require('isomorphic-fetch');
 // used when smart FAQ installed as dependency
 const uri = 'https://graphql.kiwi.com';
+const cache = new QueryResponseCache({ size: 200, ttl: 30 * 60 * 1000 });
+
+const shouldRouteCache = (queryName: string) => queryName.startsWith('FAQ');
```

Comment:
this isn't too scalable, instead, `fetchQuery` can accept the third param, `cacheConfig` with property `force`. IMHO It should be decided upon a value of this prop whether we should use cache or not 🙂 

... and then pass `cacheConfig=({force: true}}` to every `<QueryRenderer>` that should always fetch fresh data

Reply:
Absolutely agree, this was a first version of the route "whitelisting" but your suggestions sounds definitely better, thx.